docs/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+#### What does this PR do?
+
+#### Where should the reviewer start?
+
+#### Ideas that were considered and discarded
```

Comment:
I'd tag this as `[optional]`.

Reply:
Maybe
```
#### Rationale / Background / Ideas considered and rejected [optional]
```

And perhaps a 

```
#### Possible future work [optional]
```

Which would allow PR submitters to suggest follow-on work, which we could raise tickets for after appropriate discussion.